scripts/collect-assets-workflow.js

```diff
@@ -0,0 +1,45 @@
+#! /usr/bin/env node
+import { ensureDir } from "fs-extra/esm";
+import { resolve, join } from "path";
+import { cwdToProjectRoot } from "./_util.js";
+import { downloadWorkflowArtifacts, downloadAndExtractTarGz } from "./_download.js";
+import dotenv from "dotenv";
+dotenv.config();
+
+const REPO_OWNER = "konveyor";
+const REPO_NAME = "kai";
+const WORKFLOW_FILE = "build-and-push-binaries.yml";
+
+const GITHUB_API = "https://api.github.com";
+const BASE_URL = `${GITHUB_API}/repos/${REPO_OWNER}/${REPO_NAME}`;
+const DOWNLOAD_DIR = resolve("downloaded_assets");
+const META_FILE = join(DOWNLOAD_DIR, "kai", "collect.json");
+
+cwdToProjectRoot();
+
+await ensureDir(DOWNLOAD_DIR);
+
+// Download Kai assets via workflow artifacts
+await downloadWorkflowArtifacts({
+  targetDirectory: join(DOWNLOAD_DIR, "kai/"),
+  metaFile: META_FILE,
+  url: BASE_URL,
+  workflow: WORKFLOW_FILE,
```

Comment:
Suggest to make this look more like the release download function:
```js
  targetDirectory: join(DOWNLOAD_DIR, "kai/"),
  metaFile: join(DOWNLOAD_DIR, "kai", "collect.json"),

  org: "konveyor",
  repo: "kai",
  workflow: "build-and-push-binaries.yml",
```

Reply:
updated